src/Vipps.net/Infrastructure/BaseServiceClient.cs

```diff
@@ -91,19 +91,19 @@ CancellationToken cancellationToken
                 httpContent,
                 cancellationToken
             );
+#pragma warning disable IDE0079 // Remove unnecessary suppression
```

Comment:
Should the comment say why the warning is disabled?

Reply:
Yeah good idea, commenting